unpack-trees.c

```diff
@@ -1333,7 +1335,8 @@ static int clear_ce_flags_dir(struct index_state *istate,
 		rc = clear_ce_flags_1(istate, cache, cache_end - cache,
 				      prefix,
 				      select_mask, clear_mask,
-				      pl, ret);
+				      pl, ret,
+				      progress_nr);
```

Comment:
whats the convention for the line break here? When do we list them on the same line, when do we line break. And why?

Reply:
There isn't really a good reason here. It probably could have been the same line.